lib/commonmark.js

```diff
@@ -86,7 +86,7 @@ function renderHtml(markdownString, options, cb) {
 }
 
 function parse(markdownString, cb) {
-  return cb(null, commonmarkRenderer.parse(markdownString));
+  return cb(null, commonmarkRenderer.parse(markdownString, {}));
```

Comment:
It is calling it, but it is not a synchronous API call. => it doesn't make sense to be asynchronous-looking-like